java/server/src/org/openqa/selenium/docker/Docker.java

```diff
@@ -133,7 +133,7 @@ public Container create(ContainerInfo info) {
       output.write(info);
     }
 
-    LOG.info("Creating container: " + json);
+    LOG.finest("Creating container: " + json);
```

Comment:
Creating a container is an expensive and error-prone operation. Keeping the level at `info` is helpful.